src/Microsoft.Maintainability.Analyzers/UnitTests/Microsoft.Maintainability.Analyzers.UnitTests.csproj

```diff
@@ -162,6 +162,9 @@
     <Compile Include="AvoidUnusedPrivateFieldsTests.cs" />
     <Compile Include="AvoidUnusedPrivateFieldsTests.Fixer.cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
These Service include tags shouldn't be there in the csproj files. Looks like VS has added these for you for some reason? Please roll back these changes from the PR.